pkg/server/job_profiler.go

```diff
@@ -78,8 +79,20 @@ func (s *statusServer) RequestJobProfilerExecutionDetails(
 		e.addLabelledGoroutines(ctx)
 		e.addClusterWideTraces(ctx)
 
-		// TODO(dt,adityamaru): add logic to reach out the registry and call resumer
-		// specific execution details collection logic.
+		r, err := execCfg.JobRegistry.GetResumerForClaimedJob(jobID)
+		if err != nil {
+			// Request job execution details is best effort, so swallow the error
```

Comment:
Huh, why not report this back to the user?

Reply:
A few tests were failing because the resumers weren't in the registry's map, so I convinced myself its okay to present as much information as we can. But I realize this means that we'd never see errors this way. Let me think some more and see if the tests are easier to fix.

Reply:
done